en/ros/mavros_custom_messages.md

```diff
@@ -106,7 +103,7 @@ Follow *Source Installation* instructions from [mavlink/mavros](https://github.c
    rm -r common
    rm -r standard
    ```
-1. Git clone "mavlink_generator" in your home directory (or any directory you want) and execute it.
+1. Git clone "mavlink_generator" to any directory you want and execute it.
    ```sh
    git clone https://github.com/mavlink/mavlink mavlink-generator
    cd mavlink-generator
```

Comment:
@RicardoM17 Does next line actually work? The mavlink generator requires tkinter, which isn't present on most people's computers: https://mavlink.io/en/getting_started/installation.html
You'd also have to set up the PYTHONPATH.

An easier way to describe this might be to ignore the GUI and just provide the mavgen command line as copy and paste - see https://mavlink.io/en/getting_started/generate_libraries.html#mavgen

Up to you.